ansible/roles/system/tasks/main.yml

```diff
@@ -68,6 +68,36 @@
   tags:
     - touches_boot_partition
 
+- name: Comment out the original dtoverlay config.
+  replace:
+    dest: ./config.txt
+    regexp: ^(dtoverlay=vc4-kms-v3d)$
+    replace: '#\1'
+
+- name: Change dtoverlay value for DRM VC4 V3D driver.
+  replace:
+    dest: ./config.txt
+    regexp: ^(\[pi4\])$
+    replace: \1\ndtoverlay=vc4-fkms-v3d
+
```

Comment:
I don't think the above logic is idempotent.  You will keep adding `dtoverlay=vc4-fkms-v3d` to the `[pi4]` block on every run. 

I would probably either try [builtin.ini](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/ini_lookup.html) or just [builtin.lineinfile](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/lineinfile_module.html).

Reply:
I've used `lineinfile` instead. Thanks!